spec/system/kudo_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Kudo test', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+
+  it 'crud kudo' do
+    visit root_path
+    click_link 'Login page'
+    fill_in 'Email', with: employee.email
+    fill_in 'Password', with: employee.password
+    click_button 'Log in'
+    expect(page).to have_content 'Signed in successfully.'
```

Comment:
You have already covered this in signup/login tests, so I would suggest using some login helper - check out that article:
https://sajadtorkamani.com/how-to-login-in-capybara-tests-when-using-devise/